collect_app/src/main/java/org/odk/collect/android/widgets/UrlWidget.java

```diff
@@ -41,32 +42,43 @@
 @SuppressLint("ViewConstructor")
 public class UrlWidget extends QuestionWidget implements ButtonWidget {
```

Comment:
Does that still make sense to implement ButtonWidget? That was created in order to use WidgetViewUtils.createSimpleButton() - one method used in widgets for generating a button but if we are going to use xml files instead of creating those buttons programmatically we probably can get rid of it right?